rasa/run.py

```diff
@@ -36,8 +36,6 @@ def run(
     import rasa.nlu.run
     from rasa.core.utils import AvailableEndpoints
 
-    model_path = get_model(model)
```

Comment:
did we really unpack a model before in a random path and then remove this part, while unpacking it to a different path in the server? oh dear 😄

Reply:
We were!